omnigibson/robots/holonomic_base_robot.py

```diff
@@ -302,23 +307,25 @@ def _postprocess_control(self, control, control_type):
         u_vec, u_type_vec = super()._postprocess_control(control=control, control_type=control_type)
 
         # Change the control from base_footprint_link ("base_footprint") frame to root_link ("base_footprint_x") frame
-        base_orn = self.base_footprint_link.get_position_orientation()[1]
-        root_link_orn = self.root_link.get_position_orientation()[1]
+        base_orn = ControllableObjectViewAPI.get_position_orientation(self.articulation_root_path)[1]
```

Comment:
I think this is wrong? And Also, why are we calling `ControllableObjectViewAPI` directly?